scripts/install-windows-prereqs.ps1

```diff
@@ -34,6 +34,8 @@ Param(
     [string]$Python3ZipHash = 'FB65E5CD595AD01049F73B47BC0EE23FD03F0CBADC56CB318990CEE83B37761B',
     [string]$NSISURL = 'https://oejenkins.blob.core.windows.net/oejenkins/nsis-3.05-setup.exe',
     [string]$NSISHash = '1A3CC9401667547B9B9327A177B13485F7C59C2303D4B6183E7BC9E6C8D6BFDB',
+    [string]$GetPipURL = 'https://bootstrap.pypa.io/get-pip.py',
+    [string]$GetPipHash = 'CE486CDDAC44E99496A702AA5C06C5028414EF48FDFD5242CD2FE559B13D4348',
```

Comment:
Non-blocking: This is going to break us at some point; the URL points to the latest version of get-pip.py instead of a fixed version, which means we're going to get hash check failures; it's not a unique problem to this package, but have we considered targeting a fixed version, even if is not the latest version? (e.g. https://bootstrap.pypa.io/3.4/get-pip.py)

Reply:
Done